### PR TITLE
Create Demandbase Partner Directory profile

### PR DESCRIPTION
**Pull request by clarissabee**
@teeyoutan Can we please create a listing for Demandbase in the Partner Directory this week? Ideally on Wednesday 1/14 (integration and KB article should both be complete by then).

I still don't have a way to upload the logo, so I will just forward that to you via e-mail.

### CODE REVIEW

**Comment by teeyoutan** [comment]
@stewsmith  This is the one that needs to have /index.hbs add to the end of the file structure. can you fix ?

File: website/partners/technology/demandbase/index.hbs [hunk 1]
```diff
@@ -0,0 +1,36 @@
+---
+layout: partners.hbs
+title: "demandbase"
+integrations: true
+priority: 52
+summary: "Track phone calls as an offline conversion goal using ifbyphone."
```
Comment (stewsmith) [inline website/partners/technology/demandbase/index.hbs]: This is false.

**Comment by stewsmith** [comment]
@teeyoutan Updated. See my comment though.

File: website/partners/technology/demandbase/index.hbs (hunk 1 above)
Comment (teeyoutan) [inline website/partners/technology/demandbase/index.hbs]: what is? integrations?

**Comment by stewsmith** [comment]
LGTM. Let me know when that article link is live and I'll merge.

**Pull request merged**